src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -345,7 +347,7 @@ const UsersFragment = ({
     );
   return (
     <View>
-      {loadingDM && (
+      {(loadingDM || loadingCreateChat) && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here is logical as it now considers both `loadingDM` and `loadingCreateChat` states for showing the loading view. This will ensure that the loading view is displayed whenever either direct message loading or chat creation loading is in progress.
```diff
- {(loadingDM || loadingCreateChat) && (
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->